### ISSUE TITLE
Cannot find the .pst Outlook c alendar, only the .ost is available

### ISSUE THREAD

**Issue by rippey154** [phw198/OutlookGoogleCalendarSync#1009]
**Are you already using OGCS?**
Provide the version number: 2.8.0.0

**Describe the question you have**
A clear and concise description of your question and what you would like clarifying.

I have used OGCS on one laptop. I now have a new one and am trying to setup exactly the same, but I have a failure point within OGCS. 

Here is my setup: I have an IMAP and a POP3 account within Outlook, both to the same email address (yes, this means I download every message twice, and my .ost and .pst file sizes both grow).
In Outlook, I have 3 Calendars: "Calendar (This computer only)", "Calendar" and "Checkfront". I want the second one, "Calendar" to be synced via OGCS.

In OGCS, in Outlook settings, within 'Select calendar', I am only able to select "Default Calendar (This computer only)." I do not have the option to select either of the other two.
When I select "Alternative mailbox", a red bar shows underneath "Getting calendars" but nothing changes. 
When I select Shared Calendar it redirects me to the popup within Outlook to select a user, but I don't have a shared calendar so this isn't appropriate anyway.

In my previous setup on the old laptop, "Default mailbox" is selected, and the Select calendar is set to Default Calendar. Note that within this Select calendar on the old laptop, I can select Checkfront fi I want to (I do not want to), but I do not have the first option of "This computer only". I cannot figure out what is different in Outlook to cause this behavior in OGCS

Can you advise on how to set and/or find the correct default mailbox.

Thanks!

**Comment by phw198** [comment]
[This article](https://www.msoutlook.info/question/do-not-risk-losing-your-contacts-and-calendar-when-using-imap) looks pertinent to your situation maybe? 

You've not said which version of Outlook was on the previous laptop and which version you're setting up now, but the behaviour of configuring IMAP accounts after 2010 has changed regarding local calendar folders...

**Comment by rippey154** [comment]
Thank you @phw198 for your response. 

SUCCESS - because I use both POP3 and IMAP, I naturally had an .ost file and a .pst file. I had to change the default data file from .ost to the .pst file - this was the case in my 'old' laptop but not in the 'new' laptop. Once I changed the default file, accepted the warning, and restarted Outlook, then OGCS was able to select the "Default Calendar" for Outlook and sync successfully.

For completion of your question - 
My "old" outlook version and new are both MS365, with the following precision (note 32bit vs 64):
OLD
 - Version 2005 (Build 1287.20268)
 - Microsoft 365 MSO (16.0.12827.20200) 32-bit
NEW
 - Version 2005 (Build 12827.20268
 - Microsoft 365 MSO (16.0.12827.20236) 64-bit
